ppb/sprites.py

```diff
@@ -125,12 +124,40 @@ def center(self):
 
     @center.setter
     def center(self, value):
-        if self.side in (TOP, BOTTOM):
-            setattr(self.parent, self.side, value[1])
-            self.parent.center.x = value[0]
-        else:
-            setattr(self.parent, self.side, value[0])
-            self.parent.position.y = value[1]
+        self.parent.position = self._mk_update_vector_center(value)
+
+    def _mk_update_vector_side(self, attribute, value):
```

Comment:
Add a docstring documenting what this does?  It's kinda unobvious from the name or the code.